Aspire Budgeting/Views/CardView/CardExpandedView.swift

```diff
@@ -0,0 +1,100 @@
+//
+// CardExpandedView.swift
+// Aspire Budgeting
+//
+
+import SwiftUI
+
+struct CardExpandedView: View {
+
+  let cardDetails: CardDetails
+
+  private let bannerHeight: CGFloat = 100
+  private let cornerRadius: CGFloat = 24
+
+    var body: some View {
+      VStack {
+        banner
+        details
+      }
+
+    }
+}
+
+extension CardExpandedView {
+  private var banner: some View {
+    ZStack {
+      Rectangle()
+        .fill(cardDetails.bannerGradient)
+        .frame(height: bannerHeight)
+      Text(cardDetails.title)
+        .font(.nunitoBold(size: 22))
+        .foregroundColor(.white)
+    }
+  }
+
+  private var totals: some View {
+    HStack(spacing: 30) {
+
+      CardTotalsView(title: "Budgeted",
+                     amount: cardDetails.budgetedTotal,
+                     startColor: .greenFondStartColor,
+                     endColor: .greenFondEndColor,
+                     shadowColor: .greenFondShadowColor,
+                     graphImage: .minigraphUp)
+        .frame(width: 160, height: 80)
+
+      CardTotalsView(title: "Spent",
+                 amount: cardDetails.spentTotal,
+                 startColor: .redPinkFondStartColor,
+                 endColor: .redPinkFondEndColor,
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/aspirebudgetingmobile/aspirebudgeting_ios/blob/dd96be95231786735b815d5fd8f3089f90ff1b53/Aspire Budgeting/Views/CardView/CardExpandedView.swift#L50"></span>Function parameters should be aligned vertically if they’re in multiple lines in a method call.<br />
<code>vertical_parameter_alignment_on_call</code> <code>CardExpandedView.swift:50</code></td>
    </tr>
  </tbody>
</table>